Sources/GraphQLTransportWS-DataSync/Responses.swift

```diff
@@ -19,14 +21,23 @@ public struct ConnectionAckResponse: Equatable, JsonEncodable {
 
 /// A websocket `next` response from the server to the client
 public struct NextResponse: Equatable, JsonEncodable {
+    public enum UpdateType: String, JsonEncodable {
+        case flatGraphDiff
+        case recordUpdate
```

Comment:
nit: Could you document what each type is used for?

Reply:
Removed this per our conversation, flatGrapgDiffs are just all datasync diffs now.